backend/tests/test_visualization.py

```diff
@@ -1,6 +1,5 @@
 import pandas as pd
-
-from preprocessing.visualization import generate_chords
+from functions.visualization import generate_chords
 
```

Comment:
might make sense to eventually move this into a visualization package, fine for now though